hknweb/admin.py

```diff
@@ -0,0 +1,47 @@
+from django.contrib import admin
+from django.contrib.auth.models import User, Group
+from django.contrib.auth.admin import UserAdmin
+from django.conf import settings
+
+# Unregister the provided model admin
+admin.site.unregister(User)
+
+# Register out own model admin, based on the default UserAdmin
+@admin.register(User)
+class CustomUserAdmin(UserAdmin):
+    list_display = ('username', 'first_name', 'last_name', 'email', 'officer', 'candidate')
+    readonly_fields = ('officer', 'candidate')
+
+    actions = ['add_cand', 'add_officer', 'remove_cand', 'remove_officer']
+
+    def officer(self, user):
+        return 'Y' if user.groups.filter(name=settings.OFFICER_GROUP).exists() else ''
+
+    def candidate(self, user):
+        return 'Y' if user.groups.filter(name=settings.CAND_GROUP).exists() else ''
+
+    def add_cand(self, request, queryset):
+        group = Group.objects.get(name=settings.CAND_GROUP)
+        for u in queryset:
+            group.user_set.add(u)
+
+    def add_officer(self, request, queryset):
+        group = Group.objects.get(name=settings.OFFICER_GROUP)
+        for u in queryset:
+            group.user_set.add(u)
+
+    def remove_cand(self, request, queryset):
+        group = Group.objects.get(name=settings.CAND_GROUP)
+        for u in queryset:
+            group.user_set.remove(u)
+
+    def remove_officer(self, request, queryset):
+        group = Group.objects.get(name=settings.OFFICER_GROUP)
+        for u in queryset:
+            group.user_set.remove(u)
+
+    add_cand.short_description = "Add selected as candidates"
```

Comment:
Could you move this into each function? (Just to place it next to where it's relevant.)

Reply:
Inside the function? Would it work?

Reply:
Denero did it; if not inside the function then maybe next to it

Reply:
(This is entirely stylistic)

Reply:
I still kinda like this way

Reply:
It didn't work putting it in the function, I'll add it right after each function